modules/WebcastPolls/resources/webcastPollsKalturaProxy.js

```diff
@@ -126,84 +126,142 @@
 
             return defer.promise();
         },
-        transmitVoteUpdate : function(metadataId, userId, selectedAnswer,pollId)
+        transmitVoteUpdate : function(metadataId, userId, selectedAnswer,pollId ,newApi)
```

Comment:
@eitanavgil 
a better approcach is to create a different class called webcastPollsKalturaProxy.js and webcastPollsKalturaProxyLegacy.js

and instasiate either this or that according to the newAPI configururation